hi-ml-multimodal/src/health_multimodal/vlp/inference_engine.py

```diff
@@ -92,26 +98,30 @@ def convert_similarity_to_image_size(
         # TODO:
         # verify_resize_params(val_img_transforms, resize_size, crop_size)
 
+        reshaped_similarity = similarity_map.reshape(target_shape)
+        align_corners_modes = "linear", "bilinear", "bicubic", "trilinear"
+        align_corners = False if interpolation in align_corners_modes else None
```

Comment:
Why don't we use "nearest" by default and let other auxiliary plotting functions to handle interpolation? I think model inference and plotting interpolation are two different concepts and they are mixed in here by design. This would reduce the code complexity here and improve the readability

Reply:
Nearest is already the default. The issue is that, if bilinear interpolation is passed, the `align_corners` kwargs should be set. Otherwise, a warning will be shown.

Maybe interpolation should not happen in this method. What do you suggest?

Reply:
[Agreed to potentially modify this in the future instead of here]

Reply:
I agree with you regarding the last comment. We should handle it in the next PR.